tools/devops/automation/post-build-pipeline.yml

```diff
@@ -0,0 +1,53 @@
+# YAML pipeline for post build operations. 
+# This pipeline will trigger automatically after successful completion of the `build_packages` stage against the specified branches.
+
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: macios
+    source: xamarin-macios
+    trigger:
+      branches:
+      - main
+      - 6.0.*
+      stages:
+      - build_packages
```

Comment:
For those that do not know, this means that a change if the xamarin-macios branches will trigger this pipeline and it will contain all the artefacts.